cloudformation/stacks/AutoSpotting/parameters.yaml

```diff
@@ -1,3 +1,8 @@
 ---
 LambdaHandlerFunction: handler.handle
 LambdaS3Bucket: cloudprowess
+LambdaMemorySize: '256'
+ExecutionFrequency: "rate(1 minute)"
+MinOnDemandNumber: '1'
+MinOnDemandPercentage: '50'
+Regions: us-east-1,eu-west-1,ca-central-1
```

Comment:
The code is actually doing an API call to get the full list of regions and would run everywhere, this configuration is just a filter for reducing the number of regions to a smaller subset in case the user doesn't want it to run everywhere. If this value is provided, all regions not matching those in this list would be skipped at runtime.